UKHomeOffice/cerberus-service
# Cop 10471: Task details version header

## Description
Cop 10471: Task details version header using accordion

## To Test

- Run locally
- Go to http://localhost:8080/airpax/tasks/123

And you should see the following
![image](https://user-images.githubusercontent.com/1534608/164423520-406052e3-d3c2-4609-ba53-d9cbf6679e7f.png)


## Developer Checklist

\* Required

- [*] Up-to-date with main branch? \*

- [*] Does it have tests?

- [*] Pull request URL linked to JIRA ticket? \*
